<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lblTodo">
          <property name="text">
           <string>To Do</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="leftListView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lblDone">
          <property name="text">
           <string>Done</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="rightListView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/files/images/add.png</normaloff>:/files/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/files/images/remove.png</normaloff>:/files/images/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/files/images/save.png</normaloff>:/files/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/files/images/load.png</normaloff>:/files/images/load.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/resource.qrc"/>
 </resources>
 <connections/>
</ui>
